bot-2.rb

```diff
@@ -0,0 +1,40 @@
+require 'rubygems'
+require 'telegram/bot'
+require 'openweather2'
+require 'nokogiri'
+require 'curb'
+
+Openweather2.configure do |config|
+	config.endpoint = 'http://api.openweathermap.org/data/2.5/weather'
+	config.apikey = 'token'
```

Comment:
and it will work with such token? use .env file with dotenv gem to store secret data